coverage/clover.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This coverage folder should be in .gitignore

Reply:
Done!